web/angular-wallet/src/app/setup/account/create-active/seed/seed.component.html

```diff
@@ -3,7 +3,7 @@ <h2 class="seed__title">
         Generate Passphrase
     </h2>
     <div class="pin-container">
-        <div class="seed__area" (mousemove)="movement($event)"></div>
+        <div class="seed__area" (mousemove)="movement($event)" (click)="handleClick($event)"></div>
```

Comment:
by the way.... I tested creation on mobile, but touch movements are not considered, instead I needed to "tap" several dozen times.

Reply:
yeah, this makes it so you only have to tap about 5 or so times. But that should still be plenty enough given the multiplier inside `movement()`